app/views/learning/show.html.slim

```diff
@@ -13,37 +13,43 @@
 
 = render 'cms_debug'
```

Comment:
I think section partials to slice this view might be in improvement. The completed section HTML is a helper method anyway.

Reply:
I have created partials for this which can be seen in the next commit